src/wallet/wallet.ts

```diff
@@ -1529,10 +1532,17 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
 
     // Create outputs
     const outputsObj: Output[] = [];
-    // NFT transactions must have the first output as the script data
-    if (isNFT) {
-      outputsObj.push(helpers.createNFTOutput(newOptions.nftData!));
+    const dataOutputs: Output[] = [];
+    if (newOptions.data && newOptions.data.length > 0) {
+      for (const dataString of newOptions.data) {
+        dataOutputs.push(helpers.createNFTOutput(dataString));
```

Comment:
question: It seems to me that this helper will create any data output, not only NFT related ones. 

Would it be a valid suggestion AND in the scope of this PR to refactor this helper method to `createDataOutput()`?

Reply:
Refactored this so it is exactly like the fullnode facade, can you please check again?